dlf/plugins/pageview/tx_dlf_ol3.js

```diff
@@ -107,6 +107,10 @@ ol.Map.prototype.rotate = function(rotation) {
         rotate = view.getRotation() + (rotation *  Math.PI/180),
         center = view.getCenter();
 
+    if (tx_dlf_viewer.ov_view != null) {
```

Comment:
This introduces a recursion in the code. When the `ov_view` objects exists it calls the prototype method `rotate` of the `ol.Map` object, which leads again to this expression. It is better to bind the map views via openlayers events with each other. Also see the `adjustViews` method in the image manipulation control.

```javascript
var adjustViews = function(sourceView, destMap) {
			var rotateDiff = sourceView.getRotation() !== destMap.getView().getRotation();
			var resDiff = sourceView.getResolution() !== destMap.getView().getResolution();
			var centerDiff = sourceView.getCenter() !== destMap.getView().getCenter();

			if (rotateDiff || resDiff || centerDiff) {
				destMap.zoomTo(sourceView.getCenter(),
					sourceView.getZoom(), 50);
				destMap.getView().rotate(sourceView.getRotation());
			}

		}
```